File: nodejs/8 kyu/155.js
// https://www.codewars.com/kata/51c8991dee245d7ddf00000e/train/javascript

// Passed

function reverseWords(str) {
  return str.split(" ").reverse().join(" ");
}

const output = reverseWords("yoda doesn't speak like this");
console.log(output);

File: python/5 kyu/155.py
# https://www.codewars.com/kata/52449b062fb80683ec000024/train/python

# Passed

def generate_hashtag(s):
    if len(s) == 0:
        return False
    
    words = s.split()
    hashtag = "#" + "".join([word.title() for word in words])
    
    if len(hashtag) > 140:
        return False
    
    return hashtag

output = generate_hashtag('Codewars Is Nice')
print(output)